### PR TITLE
Ros2 v0.8.0 control launch

### PR DESCRIPTION
**Pull request by TakaHoribe**

### CODE REVIEW

**Comment by wep21** [comment]
I guess pass_all_args is not supported in ros2 launch and vehicle_param_file need to be passed to mpc_follower.

File: control_launch/config/vehicle_cmd_gate/vehicle_cmd_gate.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+/**:
+  ros__parameters:
+    update_rate: 10
```
Comment (wep21) [inline control_launch/config/vehicle_cmd_gate/vehicle_cmd_gate.yaml:3]: 10->10.0?

Comment (TakaHoribe) [reply control_launch/config/vehicle_cmd_gate/vehicle_cmd_gate.yaml]: fixed! thank you!

**Comment by TakaHoribe** [comment]
ready for re-review

**Comment by 0x126** [comment]
add lane_departure_checker

control.launch.xml
```
- <include file="$(find-pkg-share lane_departure_checker)/launch/lane_departure_checker.launch">
+ <include file="$(find-pkg-share lane_departure_checker)/launch/lane_departure_checker.launch.xml">
+   <arg name="vehicle_param_file" value="$(var vehicle_param_file)" />
</include>
```

packages.xml
```
+ <exec_depend>lane_departure_checker</exec_depend>
```

**Comment by TakaHoribe** [comment]
Fix lane departure checker

**Pull request closed without merging**